vllm/v1/metrics/loggers.py

```diff
@@ -353,9 +356,12 @@ def __init__(self, vllm_config: VllmConfig, engine_index: int = 0):
                         self.labelname_max_lora,
                         self.labelname_waiting_lora_adapters,
                         self.labelname_running_lora_adapters,
-                    ])
+                    ],
+                    multiprocess_mode="sum"
```

Comment:
Could you add a comment here (maybe above, as part of the "LoRA metrics" comment), explaining that this metric will not be correct when using api-server scaleout, which uses prometheus mp mode.